arm/Microsoft.AppConfiguration/configurationStores/keyValues/deploy.bicep

```
@@ -4,12 +4,12 @@ param name string
 @description('Required. Name of the value.')
 param value string
 
-@description('Required. The content type of the key-values value. Providing a proper content-type can enable transformations of values when they are retrieved by applications.')
-param contentType string
-
 @description('Conditional. The name of the parent app configuration store. Required if the template is used in a standalone deployment.')
 param appConfigurationName string
 
+@description('Optional. The content type of the key-values value. Providing a proper content-type can enable transformations of values when they are retrieved by applications.')
+param contentType string
```

If optional please add default value
```suggestion
param contentType string = ''
```

---

done, gave the param a default value